### ISSUE TITLE
Most SCAs reject NA and not a finding w/o comments - need some kind of indicator to make blank comments stand out

### ISSUE THREAD

**Issue by ibjohn** [Vulnerator/Vulnerator#109]
# Prerequisites
_Before submitting a new issue, please ensure you have completed the following (replace the space in the box with an "x" to denote that it has been completed)_

- [ ] I have ensured that I am running the latest release
- [ ] The issue is repeatable
- [ ] The issue has not already been reported
 
# Category
_Please select a category for the item being created_

- [ ] Bug
- [ ] Enhancement
- [ ] Assistance Request

# Description
_Please provide a brief synopsis of the feature request or issue; if the item being created is due to a bug, please complete the "Expected Outcome", "Actual Outcome", and "Reproduction Steps" sections as well, otherwise, check the boxes to show that it doesn't apply_

## Synopsis
_Place your description here, please! (Replace this line)_

## Expected Outcome
- [ ] N/A (Non-Bug Issue)

_What did you think was going to happen? (Replace this line)_

## Actual Outcome
- [ ] N/A (Non-Bug Issue)

_What really happened? (Replace this line)_ 

## Reproduction Steps
- [ ] N/A (Non-Bug Issue)

_How did you get Vulnerator to do that? (Replace this line)_

# Attachments
_Please provide any relevant attachments, as you see fit (e.g. screenshots); if supplying vulnerability data (e.g. CKL/Nessus files or reports), please ensure that they are sanitized of IP addresses and host names and **email** them to alex.kuchta@navy.mil - **DO NOT POST VULNERABILITY FILES HERE**_

- [ ] Vulnerator Log (Required for bugs) - this can be found at ```C:\Users\%UserName%\AppData\Roaming\Vulnerator\V6Log.txt```, where ```%UserName%``` is the user profile used to run the application
- [ ] Screenshots (Optional)
- [ ] Vulnerability Files (_**SANITIZED AND EMAILED**_)

**Comment by amkuchta** [comment]
@ibjohn is there any reason that filtering does not suit your needs?  And on which reports would you want the color coding to be?

**Comment by ibjohn** [comment]
For many of us, you are absolutely correct. 
Just thinking of just making it easier to not overlook for the users. That means less chance of me and others rejecting RARs.

**Comment by CyberSecDef** [comment]
I've found when I've generated RAR/POAMs that setting the cell background to yellow when data was missing was helpful for our team mates.  Maybe set something on the output generator to do the same for vulnerator?

-----Original Message-----

**Comment by amkuchta** [comment]
@CyberSecDef @ibjohn I can definitely look into creating some coloring options... The question is, how do we want coloring done?

- Status?
- Finding Type?
- Mitigation populated?
- ???

@Vulnerator/user-reps please feel free to weigh in!

**Comment by CyberSecDef** [comment]
I normally use like a yellow/orange on blank comments (normally just for remediations, mitigations and comments columns).

I would think the Status column should never be blank.....but that’s an important one so maybe make that red?



-----Original Message-----

**Comment by amkuchta** [comment]
@CyberSecDef I need to learn to be clearer in my conveyance of thoughts... I was thinking of highlighting Status based on the text (e.g. green for "Completed", grey for "Not Applicable", etc.).  "Finding Type" would be handled in a similar fashion (i.e. "ACAS" is green, "CKL" is blue, etc.)

**Comment by CyberSecDef** [comment]
Gotcha. That makes sense.  Doing it that way also lets you 'filter' within Excel based on color.  I like it.

-----Original Message-----

**Comment by ibjohn** [comment]
This is on hold.  What information are you waiting on?  I think your status color indicator is good.
